src/3d/qgs3daxis.cpp

```diff
@@ -108,10 +108,11 @@ bool Qgs3DAxis::eventFilter( QObject *watched, QEvent *event )
     // limit ray caster usage to the axis viewport
     QPointF normalizedPos( ( float )lastClickEvent->pos().x() / mParentWindow->width(),
                            ( float )lastClickEvent->pos().y() / mParentWindow->height() );
-#ifdef DEBUG
+
     if ( event->type() == QEvent::MouseButtonRelease )
-      qDebug() << "normalized pos:" << normalizedPos << "/ viewport:" << mAxisViewport->normalizedRect();
-#endif
+    {
+      QgsDebugMsg( QString( "QGS3DAxis: normalized pos: %1 / viewport: %2" ).arg( normalizedPos ).arg( mAxisViewport->normalizedRect() ) );
```

Comment:
```suggestion
      QgsDebugMsgLevel( QString( "QGS3DAxis: normalized pos: %1 / viewport: %2" ).arg( normalizedPos ).arg( mAxisViewport->normalizedRect() ), 2 );
```

see below -- only errors allowed on debug level 1

Reply:
Sorry @nyalldawson, I do not understand the '2' parameter stands for? Is that some kind of min debug level?

Reply:
The list of usable values is not clear for me but I will use the '2' as requested.